pkg/agent/controller/networkpolicy/priority.go

```diff
@@ -310,11 +324,11 @@ func (pa *priorityAssigner) registerConsecutivePriorities(consecutivePriorities
 func (pa *priorityAssigner) insertConsecutivePriorities(priorities types.ByPriority, updates map[types.Priority]*PriorityUpdate) error {
 	numPriorities := len(priorities)
 	pLow, pHigh := priorities[0], priorities[numPriorities-1]
-	insertionPointLow := pa.initialOFPriorityFunc(pLow, pa.isSingleTier)
-	insertionPointHigh := pa.initialOFPriorityFunc(pHigh, pa.isSingleTier)
+	insertionPointLow := pa.initialOFPriority(pLow, pa.isBaselineTier)
+	insertionPointHigh := pa.initialOFPriority(pHigh, pa.isBaselineTier)
```

Comment:
why are we passing `pa.isBaselineTier` as an argument given that it is part of the `priorityAssigner` struct?

Reply:
Refactored.